v2/contribute/decisions/session/0030-get-session-from-string.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: "0030"
+title: Add a new verifyAccessToken function that takes the token as a string
```

Comment:
On second thought, maybe a function name like getSessionManually is better? Cause:
- I don't want to use verifySession cause that corresponds to the middleware, whereas getSession is functional use (which this is)
- The postfix Manually works for even createNewSession function

Reply:
Or we could go with a postfix like "WithoutSettingInResponse". So it would be `getSessionWithoutSettingInResponse". Actually, I prefer this - what do you think?